public/docs/_examples/cb-i18n/ts/app/app.component.1.html

```diff
@@ -9,3 +9,7 @@ <h1 i18n>Hello i18n!</h1>
 <!--#docregion i18n-attribute-desc-->
 <h1 i18n="An introduction header for this sample">Hello i18n!</h1>
 <!--#enddocregion i18n-attribute-desc-->
+
+<!--#docregion i18n-title-->
+<img [src]="logo" title="Angular 2 logo">
```

Comment:
This file is a precursor to the final `app.component.html`. The `title` step hasn't occurred yet. It's ok